tests/input.js

```diff
@@ -27,7 +27,7 @@ function dec(id){
 
 // class decorators and properties
 class Foobar {
-  @dec(1);
+  @dec(1)
```

Comment:
the `;` is not in the new spec for decorators. The new preset simply enforces the new spec.